src/AppBundle/Controller/Api/AccountController.php

```diff
@@ -63,6 +64,24 @@ public function loginAction(Request $request)
         return new JsonResponse($userDto);
     }
 
+    /**
+     * @Route("api/account/logout")
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    public function logoutAction(Request $request)
+    {
+        try {
+            $this->get('security.token_storage')->setToken(null);
+            return new JsonResponse("Logout successful");
+        } catch (Exception $e) {
```

Comment:
For å kalle `Exception` direkte (uten å bruke `\Exception` altså) trengs en import.
`use Exception;`